bigchaindb/pipelines/block.py

```diff
@@ -0,0 +1,80 @@
+import rethinkdb as r
+from multipipes import Pipeline, Node
+
+from bigchaindb.pipelines.utils import ChangeFeed
+from bigchaindb import Bigchain
+
+
+class Block:
+
+    def __init__(self):
+        self.bigchain = Bigchain()
+        self.txs = []
+
+    def filter_tx(self, tx):
+        if tx['assignee'] == self.bigchain.me:
+            tx.pop('assignee')
+            return tx
+
+    def delete_tx(self, tx):
+        r.table('backlog')\
+         .get(tx['id'])\
+         .delete(durability='hard')\
+         .run(self.bigchain.conn)
+
+        return tx
+
+    def validate_tx(self, tx):
+        tx = self.bigchain.is_valid_transaction(tx)
+        if tx:
+            return tx
```

Comment:
If `tx` is `False` at this point, then `False` won't be returned; instead the default Python return value will be returned, which is `None`. Is that okay? I guess when you use `None` as a boolean expression, then it will be evaluated as if it were `False`? Maybe add a comment or docstring about this? i.e. This method returns either the transaction or `None`.